frontend/views/components/Modal/Modal.vue

```diff
@@ -6,20 +6,23 @@
 <script>
 import sbp from '~/shared/sbp.js'
 import { OPEN_MODAL, REPLACE_MODAL, CLOSE_MODAL } from '@utils/events.js'
-import { logExceptNavigationDuplicated } from '@controller/utils/misc.js'
 
 export default {
   name: 'Modal',
   data () {
     return {
       content: null, // This is the main modal
       subcontent: [], // This is for collection of modal on top of modals
-      replacement: null
+      replacement: null,
+      isUrlChange: true
```

Comment:
The comments you have for `content` and `subcontent` are nice; can you add similar comments for `replacement` and `isUrlChange`?